README.md

```
@@ -1,203 +1,260 @@
+- 외부 광고 API 인증은 `AdAuthFactory` + `AdAuthStrategy`로 추상화해 플랫폼 추가 시 전략만 확장합니다.
+- 외부 HTTP 호출은 Feign / WebClient로 일원화하고, 실패 가능 구간은 Spring Retry로 감쌉니다.
+- 토큰·시크릿 등 민감 정보는 `AESUtil`로 암호화해 저장합니다.
+- 목록 조회 시 무한 스크롤은 `CursorUtil` 기반 커서 페이지네이션을 사용합니다.
+
+<br>
+
+## 👥 Contributors
+
+| <div align="center">[김지민](https://github.com/jinnieusLab)</div>                                    | <div align="center">[김민규](https://github.com/kingmingyu)</div>                                    | <div align="center">[오준영](https://github.com/ojy0903)</div>                                    |
+| ----------------------------------------------------------------------------------------------------- | ---------------------------------------------------------------------------------------------------- | -------------------------------------------------------------------------------------------------- |
+| <div align="center"><img src="https://avatars.githubusercontent.com/jinnieusLab" width="160" /></div> | <div align="center"><img src="https://avatars.githubusercontent.com/kingmingyu" width="160" /></div> | <div align="center"><img src="https://avatars.githubusercontent.com/ojy0903" width="160" /></div> |
```

markdownlint-cli2 (0.23.1)

[warning] Images should have alternate text (alt text)

(MD045, no-alt-text)